src/client/components/AddNewRoom/RoomHandler.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+import AddRoom from './AddRoom';
+
+const RoomHandler = () => {
+  const [newRoom, setNewRoom] = useState('');
```

Comment:
I would suggest using `roomName` instead of `newRoom` since it is more clear that way.